src/target/qemu/Makefile.inc

```diff
@@ -0,0 +1,47 @@
+FILESYSTEMS := common base_fonts 320x240x16
+
+SCREENSIZE  := 320x240x16
+FONTS        = filesystem/$(FILESYSTEM)/media/15ascii.fon \
+               filesystem/$(FILESYSTEM)/media/23bold.fon
+
+ifndef BUILD_TARGET
+CROSS    = arm-none-eabi-
+
+ALL = $(LIBOPENCM3) $(TARGET).bin
+
+NUM_MODELS ?= 10
+
+LINKFILE = $(SDIR)/target/$(TARGET)/$(TARGET).ld
+LIBOPENCM3 = $(SDIR)/libopencm3/lib/libopencm3_stm32f1.a
+
+SRC_C   := $(wildcard $(SDIR)/target/$(TARGET)/*.c) \
```

Comment:
I think we should create a Makefile.inc for all STM32 related parts. this makefile, the one under devo/common, and the one under at9 are largely duplicatd.

Reply:
Yes probably.  This was only meant to be a proof-of-concept.  certainly not ready for inclusion

Reply:
we also need an instruction on qemu as well.

Reply:
I added some documentation now.